pages/pipelines/_build_states.md.erb

```diff
@@ -1,2 +1,6 @@
 Build state can be one of `creating`, `scheduled`, `running`, `passed`, `failed`, `blocked`, `canceling`, `canceled`, `skipped`, `not_run`.
 You can query for `finished` builds to return builds in any of the following states: `passed`, `failed`, `blocked`, or `canceled`.
+
+<div class="Docs__note">
```

Comment:
What about saying something like

> When a [triggered build](LINK) fails, the step that triggered it will be stuck in the `running` state forever.

Reply:
That's a pretty ominous way of saying that. I like it! :D

Reply:
:metal: